### PR TITLE
EY-3784 Selftest for søknadene

### PR DESCRIPTION
**Pull request by sebassonav**

### CODE REVIEW

File: docker/frontend.Dockerfile [hunk 1]
```diff
@@ -1,2 +1,2 @@
-FROM europe-north1-docker.pkg.dev/nais-management-233d/etterlatte/pensjon-etterlatte-etterlatte-node-server:main
+FROM europe-north1-docker.pkg.dev/nais-management-233d/etterlatte/pensjon-etterlatte-etterlatte-node-server:2024.04.29-12.46-8642cad
```
Comment (emilps) [inline docker/frontend.Dockerfile:1]: Denne skal vell fortsatt være main?

**Review by emilps — APPROVED** [review]

File: apps/etterlatte-node-server/src/selftestRouter.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import express from "express";
+import config from "./config";
+import logger from "./monitoring/logger";
+
+export const selftestRouter = express.Router()
+
+selftestRouter.get('/', express.json(), async (req, res) => {
+    try {
+        logger.info('selftesqrqwwqt')
```
Comment (emilps) [inline apps/etterlatte-node-server/src/selftestRouter.ts:9]: qeqweqeqw

Comment (sebassonav) [reply apps/etterlatte-node-server/src/selftestRouter.ts:9]: https://github.com/navikt/pensjon-etterlatte/pull/1219/commits/8928fcecc244d183fa50464b69cd46530473216d

File: docker/frontend.Dockerfile (hunk 1 above)
Comment (sebassonav) [reply docker/frontend.Dockerfile]: https://github.com/navikt/pensjon-etterlatte/pull/1219/commits/8928fcecc244d183fa50464b69cd46530473216d

**Pull request merged**